templates/files/k8s-resource/calico-policy-only.yaml

```diff
@@ -504,6 +504,8 @@ spec:
               value: "info"
             - name: FELIX_HEALTHENABLED
               value: "true"
+            - name: FELIX_PROMETHEUSMETRICSENABLED
+            value: "true"  
```

Comment:
please test your changes properly before merging, I was  testing something I need to spent 2 hours hunting this wrong indentation

k8s-addons never finished
```
 parsing /srv/calico-policy-only.yaml: error converting YAML to JSON: yaml: line 130: did not find expected '-' indicator
```

Reply:
I apologize. I tested it on godsmack so only the calico-all got applied and not this one as this is only for AWS installations. I will be more careful